### ISSUE TITLE
Make the little diamond appear on the mods button only when there are updates available for some mods

### ISSUE THREAD

**Issue by Blayung** [Prospector/ModMenu#603]

**Comment by haykam821** [comment]
Are you asking for configuration to only check for updates on a certain set of mods, rather than all loaded mods?

**Comment by Blayung** [comment]
Nope. Idk if it's intended, but for me the litte diamond appears no matter if I've got any updates available or not.

**Comment by Blayung** [comment]
Alright, I think it should be a bug report rather than an enhancement request. I see that you have already tried to make it work this way:
```
if (ModMenuConfig.BUTTON_UPDATE_BADGE.getValue() && ModMenu.modUpdateAvailable) {
    UpdateAvailableBadge.renderBadge(DrawContext, this.width + this.getX() - 16, this.height / 2 + this.getY() - 4);
}
```
(`&& ModMenu.modUpdateAvailable`)

**Comment by LostLuma** [comment]
You might not see which mod has an update as it's either hidden due to being a library and you having that setting on, or it's in one of the dropdowns from being marked as a parent of another mod. If you can't find it this way you can also look in your Minecraft log, it will have one or more lines like this: `Update available for 'dynamicfps@2.3.0', (-> 2.4.0)`

**Comment by Blayung** [comment]
Oh, well. It does not appear now! Whatever.
